package.json

```diff
@@ -123,6 +123,7 @@
     "jsdom": "^16.4.0",
     "lint-staged": "^10.4.2",
     "markdown-loader": "^6.0.0",
+    "no-mouse-days": "^0.0.3",
```

Comment:
The package is required only for development mode, so it can reside in `devDependencies`.